src/mesh.cpp

```diff
@@ -104,11 +104,33 @@ void scaleMesh(scene::IMesh *mesh, v3f scale)
 	for(u16 j=0; j<mc; j++)
 	{
 		scene::IMeshBuffer *buf = mesh->getMeshBuffer(j);
-		video::S3DVertex *vertices = (video::S3DVertex*)buf->getVertices();
-		u16 vc = buf->getVertexCount();
-		for(u16 i=0; i<vc; i++)
+		const video::E_VERTEX_TYPE vType = buf->getVertexType();
+		switch(vType)
```

Comment:
Needs a space between switch and (vType).  Also opening brace needs to be on the same line.